commands/pull.py

```diff
@@ -0,0 +1,59 @@
+# from _typeshed import NoneType
+# from commands import create
+import re
+from functions import create_temp_vc
+from discord.user import User
+from commands.base import Cmd
+
+help_text = [
+    [
+        ("Usage:", "<PREFIX><COMMAND> [@MEMBERS]"),
+        ("Description:",
+         "Try to pull another member into a temporary channel"),
+        ("Example:", 
+        "```<PREFIX><COMMAND> @Stonic#7341```\nFor Multiple drags: ```<PREFIX><COMMAND> @Stonic#7341 @Zelda#4567```"),
+    ]
+]
+
+async def execute(ctx, params):
+    user: User = User
+    guild = ctx['guild']
+    client = ctx['client']
+    author = ctx['message'].author
+    users = params
+
+    members = []
+
+    for p in users:
+        try:
+            user = re.sub(r'[<>@!]', '', p)
+            print(f"incoming: {p}, user: {user}")
+
+            cm = await guild.fetch_member(user)
+        except:
+            return False, f"Member {p} wasn't found in this server."
+
+        members.append(cm)
+
+    vc = await create_temp_vc(client, guild, author, members)
+
+    print(f"{vc}")
+
+    if vc == None:
```

Comment:
Needs to be handle, to check rather if it's `None` or If the channel already exists.

Reply:
Is there still something to work on here?